### PR TITLE
feat: extend cafs object

### PR DESCRIPTION
**Pull request by d3lm**
This PR extends the cafs object with `cafsDir` and `getFilePathInCafs`. Furthermore, some types have been moved into a separate `@pnpm/cafs-types` package.

### CODE REVIEW

File: packages/cafs-types/README.md [hunk 1]
```diff
@@ -0,0 +1,28 @@
+# @pnpm/types
```
Comment (zkochan) [inline packages/cafs-types/README.md]: the name is not correct

Comment (d3lm) [reply packages/cafs-types/README.md]: Oh good catch!

Comment (d3lm) [reply packages/cafs-types/README.md]: Fixed.

File: packages/create-cafs-store/src/index.ts [hunk 2]
```diff
@@ -80,6 +80,7 @@ export default function createCafsStore (
   })
   return {
     ...createCafs(cafsDir, opts?.ignoreFile),
+    cafsDir,
```
Comment (zkochan) [inline packages/create-cafs-store/src/index.ts:83]: do you still need this? Now getFilePathInCafs is exported.

Comment (d3lm) [reply packages/create-cafs-store/src/index.ts:83]: Yes, I still need it, because I need to know where to link the package to.

**Review by zkochan — APPROVED** [review]

**Pull request merged**